lms/startup.py

```diff
@@ -77,7 +77,7 @@ def enable_theme():
     # Workaround for setting THEME_NAME to an empty
     # string which is the default due to this ansible
     # bug: https://github.com/ansible/ansible/issues/4812
-    if settings.THEME_NAME == "":
+    if getattr(settings, "THEME_NAME", "") == "":
```

Comment:
Why are we defaulting this to a string? If it's already `None`, we
shouldn't reset it, right?

```python
if getattr(settings, 'THEME_NAME') == '':
    settings.THEME_NAME = None
```

Further, what prompted the change to `getattr`; why is `THEME_NAME` now
optional?